src/parcsr_ls/par_lr_interp_device.c

```diff
@@ -1017,7 +1030,7 @@ hypreDevice_extendWtoP( HYPRE_Int      P_nr_of_rows,
 // For Ext+i Interp, scale AFF from the left and the right
 __global__
 void compute_twiaff_w( HYPRE_Int      nr_of_rows,
```

Comment:
Should this also be a hypreCUDAKernel ?

Reply:
@ulrikeyang Yes, it should be, and has been fixed. The idea is to prefix our kernels with `hypreCUDAKernel_`. There are still other places we missed, which I will change later. Thanks!

Reply:
@ulrikeyang I missed `par_2s_interp_device.c`, which has been pushed.